deployments/cloudfoundry/buildpack/README.md

```diff
@@ -0,0 +1,168 @@
+    1) - `SPLUNK_INGEST_URL` - The ingest base URL for Splunk. This option takes precedence over SPLUNK_REALM.
+       - `SPLUNK_API_URL` - The API server base URL for Splunk. This option takes precedence over SPLUNK_REALM.
+    2) - `SPLUNK_REALM` - The Splunk realm in which your organization resides. Used to derive SPLUNK_INGEST_URL
+       and SPLUNK_API_URL.
+
+Optional:
+- `OS` - Operating system that Cloud Foundry is running. Must match format of Otel Collector executable name.
+    Default: `linux_amd64`
+- `OTEL_CONFIG` - Local name of OpenTelemetry config file. Default: `otelconfig.yaml`
+- `OTEL_VERSION` - Executable version of OpenTelemetry Collector (contrib) to use. The buildpack depends on features present in version
+    v0.48.0+. Default: `latest`. Example valid value: `v0.48.0`.
+    Note that if left the default value, the latest version will be found and later variable references will be
+    to a valid version number, not simply the word "latest".
+- `OTEL_BINARY` - OpenTelemetry Collector executable file name. Default: `otelcontribcol_$OS-v$OTEL_VERSION`
+- `OTEL_BINARY_DOWNLOAD_URL` - URL to download the OpenTelemetry Collector from. This takes precedence over other
+    version variables. Only the Splunk distribution is supported.
+    Default: `https://github.com/signalfx/splunk-otel-collector/releases/download/${OTEL_VERSION}/otelcol_${OS}`
+- `RLP_GATEWAY_SHARD_ID` - Metrics are load balanced between receivers that use the same shard ID.
+   Only use if multiple receivers must receive all metrics instead of
+   balancing metrics between them. Default: `otelcol`
+- `RLP_GATEWAY_TLS_INSECURE` - Whether to skip TLS verify for the RLP gateway endpoint. Default: `false`
+- `UAA_TLS_INSECURE` - Whether to skip TLS verify for the UAA endpoint. Default: `false`
+
+## Sidecar Configuration
+
+The recommended method for running the Collector is to run it as a sidecar using
+the Cloud Foundry [sidecar
+functionality](https://docs.cloudfoundry.org/devguide/sidecars.html).
+Additional information can be found [in the v3 API
+docs](http://v3-apidocs.cloudfoundry.org/version/release-candidate/#sidecars).
+
+Here is an example application `manifest.yml` file that would run the Collector as
+a sidecar:
+
+```yaml
+---
+applications:
+  - name: test-app
+    buildpacks:
+      - otel_collector_buildpack
+      - go_buildpack
+    instances: 1
+    memory: 256M
+    random-route: true
+    env:
+      RLP_GATEWAY_ENDPOINT: "https://log-stream.sys.<TAS environment name>.cf-app.com"
+      UAA_ENDPOINT: "https://uaa.sys.<TAS environment name>.cf-app.com"
+      UAA_USERNAME: "..."
+      UAA_PASSWORD: "..."
+      SPLUNK_ACCESS_TOKEN: "..."
+      SPLUNK_REALM: "..."
+    sidecars:
+      - name: otel-collector
+        process_types:
+          - web
+        command: "$HOME/../deps/otelcontribcol_${OS:-linux_amd64}-v${OTEL_VERSION:-0.47.0} --config=$HOME/../deps/${OTEL_CONFIG:-otelconfig.yaml}"
```

Comment:
```suggestion
        command: "$HOME/../deps/otelcontribcol_${OS:-linux_amd64}-v${OTEL_VERSION:-0.48.0} --config=$HOME/../deps/${OTEL_CONFIG:-otelconfig.yaml}"
```

Reply:
Good find, thanks! I've updated to be 0.48.0 now.